doc/source/data/inspecting-data.rst

```diff
@@ -151,46 +151,57 @@ Inspecting execution statistics
 Ray Data calculates statistics during execution for each operator, such as wall clock time and memory usage.
 
 To view stats about your :class:`Datasets <ray.data.Dataset>`, call :meth:`Dataset.stats() <ray.data.Dataset.stats>` on an executed dataset. The stats are also persisted under `/tmp/ray/session_*/logs/ray-data.log`.
+For more on how to read this output, see :ref:`Monitoring Your Workload with the Ray Data Dashboard <monitoring-your-workload>`.
```

Comment:
later after we add the specific subsection on that page, let's be sure to link it to the specific subsection.

Reply:
Yeah makes sense, will leave for that PR.